cli/pcluster/config/validators.py

```diff
@@ -49,6 +49,8 @@
         "unsupported_architecture": "FSX Lustre can be used only with instance types and AMIs that support these "
         "architectures: {supported_architectures}. Please double check the 'master_instance_type', "
         "'compute_instance_type' and/or 'custom_ami' configuration parameters.",
+        "unsupported_backup_param": "When restoring file system from backup, {name} {value} "
```

Comment:
One more nitpick on this: instead of saying `restoring file system from backup` can you say `restoring an FSx Lustre file system from backup`? The motivation behind this request is to avoid ambiguity between the other file system types we support.